backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -604,17 +609,23 @@ public static String convertSexToDatabaseValue(String biologicalSex) {
   }
 
   private static List<FeedbackMessage> validateSpecificValueOrSNOMED(
-      ValueOrError input, Set<String> acceptableValues) {
+      ValueOrError input, Set<String> acceptableStringLiterals, Set<String> acceptableSNOMEDS) {
     List<FeedbackMessage> errors = new ArrayList<>();
     String value = parseString(input.getValue());
+
     if (value == null) {
       return errors;
     }
     boolean nonSNOMEDValue = value.matches(ALPHABET_REGEX);
+    boolean snomedValue = value.matches(SNOMED_REGEX);
+
     if (nonSNOMEDValue) {
```

Comment:
This logic wouldn't error for some invalid test result values that have been used in prod recently e.g. (`Positive, 10828004` and `-`).
<img width="643" alt="Screenshot 2024-07-24 at 10 13 34" src="https://github.com/user-attachments/assets/b029cfe1-2105-455e-80fb-dcbb4cd5fbd6">

Reply:
@emyl3 From my understanding, the point of this ticket is to make it such that `10828004` and `Positive` are supported values. I did not consider non letter chars in the filtering though -- will update that now.